pkg/apis/core/types.go

```diff
@@ -5538,6 +5538,10 @@ type NodeSystemInfo struct {
 	Architecture string
 	// Swap Info reported by the node.
 	Swap *NodeSwapStatus
+	// Whether the node is running in a user namespace.
+	// +featureGate=KubeletInUserNamespace
+	// +optional
+	RunningInUserNS *bool
```

Comment:
sorry, I didn't catch this in KEP review. Typically a new API field will mean staying in alpha for another release. I think it may be the most reasonable thing here given where we are in release cycle. Do you think you can update the target stage for this KEP?

Reply:
> Typically a new API field will mean staying in alpha for another release.

I hope that this general rule doesn't necessarily apply for adding a single boolean to a status object.